sphinx/writers/latex.py

```diff
@@ -1513,30 +1524,33 @@ def visit_entry(self, node):
                        '\\vbox{\\hbox{\\strut}}\\end{varwidth}%\n') + context
             self.needs_linetrimming = 1
         if len(node.traverse(nodes.paragraph)) >= 2:
-            self.table.has_oldproblematic = True
+            for i in range(cell.col, cell.col + cell.width):
```

Comment:
goal here is that multi-paragraph cells must be rendered in p-type columns (``\Y`` or ``\X`` but not ``l``)

Reply:
I think `LaTeXTranslater` should not access `table.empirical_colwidths` directly.
I think encapsulation is good technique for boundary of responsibility.
How about `table.update_colwidth(cell, bodywidthpercol=None)`?